cmake/modules/HCT.cmake

```diff
@@ -87,6 +107,7 @@ function(add_hlsl_hctgen mode)
                      COMMAND ${PYTHON_EXECUTABLE}
                              ${hctgen} ${force_lf}
                              ${mode} --output ${temp_output} ${input_flag}
+                     ${format_cmd}
```

Comment:
If there's no clang format, I don't expect this to end well?

Reply:
If a variable in CMake is not set it resolves to an empty string.